crates/eurocode_1_1_2/src/section_3/equation_3_2.rs

```diff
@@ -0,0 +1,26 @@
+pub fn net_convective_heat_flux_surface(alpha_c: f64, delta_g: f64, delta_m: f64) -> f64 {
+    alpha_c * (delta_g - delta_m)
```

Comment:
```suggestion
pub fn net_convective_heat_flux_surface(alpha_c: f64, theta_g: f64, theta_m: f64) -> f64 {
    alpha_c * (theta_g - theta_m)
```

Same comment for the rest of this document. Replace `delta` with `theta`

Reply:
I need to learn my greek symbols ....